packages/backend/src/app/database/postgres-connection.ts

```diff
@@ -159,6 +160,7 @@ const getMigrations = (): (new () => MigrationInterface)[] => {
                 AddPlatform1697717995884,
                 AddCustomDomain1698077078271,
                 AddSigningKey1698602417745,
+                ManagedAuthnInitial1698700720482,
```

Comment:
Let's move to all platforms specially the edits on project and user

Reply:
done